exercises/pascals-triangle/src/example.h

```diff
@@ -0,0 +1,9 @@
+#ifndef PASCALS_TRIANGLE_H
+#define PASCALS_TRIANGLE_H
+#include <stdlib.h>
+#include <string.h>
+
+void pt_free(size_t **, size_t);
```

Comment:
I might just be dense, but it took me a little bit to figure out what `pt_free` meant. How about making this match the other function? Perhaps `create_triangle`/`generate_triangle` and `free_triangle`?